driver-core/src/test/unit/com/mongodb/internal/connection/ScramShaAuthenticatorSpecification.groovy

```diff
@@ -445,16 +427,17 @@ class ScramShaAuthenticatorSpecification extends Specification {
         assert(clientMessages.size() == sent.size())
         sent.indices.each {
             def sentMessage = sent.get(it)
-            def messageStart = it == 0 ? "saslStart: 1, mechanism:'$mechanism', options: {skipEmptyExchange: 1}"
+            def messageStart = it == 0 ? "saslStart: 1, mechanism:'$mechanism', options: {skipEmptyExchange: true}"
                     : 'saslContinue: 1, conversationId: 1'
             def expectedMessage = BsonDocument.parse("{$messageStart, payload: BinData(0, '${encode64(clientMessages.get(it))}')}")
             assertEquals(expectedMessage, sentMessage)
         }
     }
 
-    def validateAuthentication(String payloads, ScramShaAuthenticator authenticator, boolean async) {
-        def (clientMessages, serverResponses) = createMessages(payloads)
-        def connection = createConnection(serverResponses)
+    def validateAuthenticationWithAndWithoutEmptyExchange(String payloads, ScramShaAuthenticator authenticator,
```

Comment:
Nit: the current name of the method is fine.  The extra parameters don't change that.

Reply:
I have reverted the method name.